app/managers.py

```diff
@@ -0,0 +1,39 @@
+import os
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect(os.path.join("..", "cinema"))
```

Comment:
Cinema db can be created in any directory. I think you can just write `sqlite3.connect("cinema")` to make it simple)